packages/components/src/internal/components/assay/SpecialtyAssayPanel.tsx

```diff
@@ -31,27 +30,39 @@ export const SpecialtyAssayPanel: FC<SpecialtyAssayPanelProps> = memo(props => {
         [onChange]
     );
 
+    const noPremiumClass = useMemo(() => {
+        if (children)
+            return "large-margin-top";
+
+        if (warning)
+            return "";
+
+        return "margin-top";
+    }, [children, warning])
+
     return (
         <div>
             <Row>
                 <Col xs={6}>
                     <div className="margin-bottom">
                         <b>Use Instrument Specific Data Format</b>
                     </div>
-                    <div className="margin-bottom">
-                        {selected && options && (
-                            <select
-                                id="specialty-assay-type-select"
-                                value={selected.name}
-                                onChange={onSelectChange}
-                                className="form-control"
-                            >
-                                {options}
-                            </select>
-                        )}
-                    </div>
+                    {selected && options?.length > 0 && (
```

Comment:
This will hide the select input if there are no options / assay providers, but it will still show the "Use Instrument Specific Data Format" heading. I haven't looked at the mockups, is that wanted? or should this conditional be moved up a bit?

Reply:
Yeah you're right.  I had some conflicting info on that. Updated.